src/components/common/CopyButton/index.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { useEffect, useState } from 'react'
+import styled from 'styled-components'
+import { faCopy } from '@fortawesome/free-regular-svg-icons'
+import { faCheck } from '@fortawesome/free-solid-svg-icons'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import CopyToClipboard from 'react-copy-to-clipboard'
+
+import { DISPLAY_TEXT_COPIED_CHECK } from 'apps/explorer/const'
+
+const Icon = styled(FontAwesomeIcon)<{ copied: boolean }>`
```

Comment:
I'm fined with this name, but I want to make sure we are consistent. Do we have a naming convention for the styled components props. Should it be `$copied` or `copied`?

 @W3stside?

Reply:
@alfetopito Appreciate the fact you implemented this, by going the extra mile :) - Only thing to add is to show 'copied' next the to green check on click (Like Etherscan) for explicit textual validation.

Reply:
> Should it be $copied or copied?

Well, not a big fan of `$` (jQuery feelings).
Anyway, it's an internal prop, so it's explicit enough.
Maybe if it was an exposed component it would be better do differentiate the props, but I don't see the need here.

> Only thing to add is to show 'copied' next the to green check on click (Like Etherscan) for explicit textual validation.
Sure, will add it

Reply:
$(thankYou) $(veryMuch)

Reply:
i think we decided a little while ago (too long to remember so not pointing fingers) that we are dropping the styled `$` convention as it's not super helpful AND clashes with `styled-theming`